### DISCUSSION TITLE
Add vegetable cutlets!

### DISCUSSION THREAD

**Discussion by collectanos** [cataclysmbn/Cataclysm-BN#2058] [General]
Adding carrot cutlets, squash cutlets and etc!
Below recipes are studied by the book Mommy Recipes.(cooking in any ovens ). Carrot cutlet-1 flour (plain,except for bone ), 2 eggs (any),1 carrot .(slightly restores satiety and slightly reduces thirst) (3 minutes); Squash cutlets(2 servings) - flour (plain,except for bone),3 eggs(any),1 vegetable marrow(12 minute)(you eat well, but you want to drink decently); Draniki- 1 flour (plain),1eggs,2 potates(2 minutes)(weak snack like) Book Mommy Recipes (can be found in the kitchen in houses 17% and you need level 2 cooking to read)

**Comment by Relgar** [comment]
Please give a bit more of a specific description into what these are, and a recipe (perhaps from existing foodstuffs, using existing tools) of how one might make this. Would this be a common recipe that everyone should autolearn, or is this something they should learn from a book with more in-game lore etc.

In short, please flesh out this idea instead of saying 'add x y z'.

**Comment by collectanos** [comment]
Below recipes are studied by the book Mommy Recipes.(cooking in any ovens )
Carrot cutlet-1 flour (plain,except for bone ), 2 eggs (any),1 carrot .(slightly restores satiety and slightly reduces thirst) (3 minutes)
Squash cutlets(2 servings)  - flour (plain,except for bone),3 eggs(any),1 vegetable marrow(12 minute)(you eat well, but you want to drink decently)
Draniki- 1 flour (plain),1eggs,2 potates(2 minutes)(weak snack like) 
Book Mommy Recipes (can be found in the kitchen in houses 17% and you need level 2 cooking to read)

**Comment by Relgar** [comment]
It appears to be vegetarian in nature - is there a particular reason for this suggestion? Is there a niche that current vegetarian foods do not fit, or does this help some part of a vegetarian run?

I'm trying to think of a reason we would include this aside from "it's cool", in which case I'd suggest making a mod instead of getting it pulled into the base game.
